packages/vision/package.json

```diff
@@ -53,8 +53,7 @@
     "arrify": "^1.0.0",
     "async": "^1.4.2",
     "extend": "^3.0.0",
-    "@google-cloud/common": "^0.1.0",
-    "@google-cloud/storage": "^0.1.0",
```

Comment:
I'm assuming there is a system test for this? Don't we still need it as a dev dependency?